<ui version="4.0" >
 <class>UISettingsAStyle</class>
 <widget class="QWidget" name="UISettingsAStyle" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AStyle Settings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>3</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabStyle" >
      <attribute name="title" >
       <string>Style</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>5</number>
       </property>
       <property name="topMargin" >
        <number>5</number>
       </property>
       <property name="rightMargin" >
        <number>5</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gbStyles" >
         <property name="title" >
          <string>Styles</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>3</number>
          </property>
          <property name="leftMargin" >
           <number>5</number>
          </property>
          <property name="topMargin" >
           <number>5</number>
          </property>
          <property name="rightMargin" >
           <number>5</number>
          </property>
          <property name="bottomMargin" >
           <number>5</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbANSI" >
            <property name="text" >
             <string>ANSI</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbKR" >
            <property name="text" >
             <string>K&amp;&amp;R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbLinux" >
            <property name="text" >
             <string>Linux</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbGNU" >
            <property name="text" >
             <string>GNU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbJava" >
            <property name="text" >
             <string>Java</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbCustom" >
            <property name="text" >
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>31</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbExample" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Sample</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>3</number>
          </property>
          <property name="leftMargin" >
           <number>5</number>
          </property>
          <property name="topMargin" >
           <number>5</number>
          </property>
          <property name="rightMargin" >
           <number>5</number>
          </property>
          <property name="bottomMargin" >
           <number>5</number>
          </property>
          <item>
           <widget class="QTextEdit" name="teSample" >
            <property name="contextMenuPolicy" >
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="acceptDrops" >
             <bool>false</bool>
            </property>
            <property name="undoRedoEnabled" >
             <bool>false</bool>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="html" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIndent" >
      <attribute name="title" >
       <string>Indentation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>5</number>
       </property>
       <property name="topMargin" >
        <number>5</number>
       </property>
       <property name="rightMargin" >
        <number>5</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labIndentSize" >
           <property name="text" >
            <string>Indentation size (in spaces):</string>
           </property>
           <property name="buddy" >
            <cstring>spnIndentation</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnIndentation" >
           <property name="value" >
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkUseTab" >
         <property name="text" >
          <string>Use TABs instead of spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkForceUseTabs" >
         <property name="text" >
          <string>Force using TABs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkConvertTabs" >
         <property name="text" >
          <string>Convert TABs to spaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFillEmptyLines" >
         <property name="text" >
          <string>Fill emty lines with the whitespace of their previous lines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentClasses" >
         <property name="text" >
          <string>Indent classes (keywords public:, protected: and private:)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentSwitches" >
         <property name="text" >
          <string>Indent switches (keyword case:)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentCase" >
         <property name="text" >
          <string>Indent case: statement in switches (command under case:)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentBrackets" >
         <property name="text" >
          <string>Indent brackets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentBlocks" >
         <property name="text" >
          <string>Indent blocks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentNamespaces" >
         <property name="text" >
          <string>Indent namespaces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentLabels" >
         <property name="text" >
          <string>Indent labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkIndentPreprocessor" >
         <property name="text" >
          <string>Indent multi-line preprocessor definitions</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFormat" >
      <attribute name="title" >
       <string>Formatting</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>3</number>
       </property>
       <property name="leftMargin" >
        <number>5</number>
       </property>
       <property name="topMargin" >
        <number>5</number>
       </property>
       <property name="rightMargin" >
        <number>5</number>
       </property>
       <property name="bottomMargin" >
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>3</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labBreakType" >
           <property name="text" >
            <string>Brackets (None, Break, Attach or Linux):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbBreakType" >
           <item>
            <property name="text" >
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Break</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Attach</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Linux</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkBreakBlocks" >
         <property name="text" >
          <string>Pad empty lines around header blocks (e.g. 'if', 'while'...)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkBreakElseIfs" >
         <property name="text" >
          <string>Break 'else if()' header combinations into separate lines</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkPadOperators" >
         <property name="text" >
          <string>Insert space padding around operators</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkPadParens" >
         <property name="text" >
          <string>Insert space padding around parenthesies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkKeepComplex" >
         <property name="text" >
          <string>Don't break complex statements and multiple statements residing in a single line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkKeepBlocks" >
         <property name="text" >
          <string>Don't break one-line blocks</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
